media/memory-inspector.css

```diff
@@ -129,34 +99,62 @@
 }
 
 .advanced-options-toggle {
-  cursor: pointer;
-  display: flex;
-  flex-flow: row nowrap;
-  align-items: center;
+  margin-left: auto;
+}
+
+.advanced-options-toggle:after { content: ''; position: absolute; width: 30px; height: 30px; transform: translateX(-50%) translateY(-25%); }
+.advanced-options-content {
+  position: absolute;
+  z-index: 1;
+  margin: 20px -165px;
```

Comment:
This `-165px` seems to depend on the new `160px` width of the `multi-select` class, since that's what forces the whole `advanced-options-content` div to its `174px` total width. It would likely be preferable to have both width and margin specified here.